text/0000-roadmap-2021.md

```diff
@@ -38,6 +38,18 @@ However, at minimum, they should provide answers to these questions:
 * Where does this group work?
 * Who is the point of contact for questions on the state of this group?
 
+The core team, upon merging of this RFC, will prepare a new repository for
+governance RFC work. This repository will have empty charters for the existing
+known groups, and changes will be approved by the core team. It is expected that
+teams forming new groups after the merge of this RFC will prepare charters and
+merge them into that repository before kicking off the group's tasks.
```

Comment:
i'm assuming the approval process is more or less the same as a regular RFC?

Reply:
Yes, I'd expect a similar or identical process.